### DISCUSSION TITLE
One Book From Series Not Appearing In Library

### DISCUSSION THREAD

**Discussion by johnhwright** [advplyr/audiobookshelf#2409] [Q&A]
Have hundreds of books all in the same location and they all work great. When adding a recent series all books in the series added correctly except the very first book. Looking at the log I see:

```
2023-12-11 09:02:57.778
ERROR
[fileUtils] Recurse files error Error: ENOTDIR: not a directory, scandir 'C:\AudioBookshelf\AudioBooks\OpenAudible_Everyone\The Book Title.m4b'
```

There doesn't seem to the be anything special about he file or filename, it's not the longest name compared to a bunch of the other working ones, and it doesn't contain any characters the others don't either.

Any ideas what might be causing this one book to fail?

**Comment by advplyr** [comment]
Did you move that book from inside a subfolder to the root folder?

**Reply by johnhwright** [reply]
So this for my audible library so it's just a single directly that open audible dumps all the m4bs into. So there are no folders and hundreds of books, and it's worked great except for now. I guessing that adding a directory around the one book would probably fix it (based on [this](https://www.audiobookshelf.org/docs#book-title-folder-naming)), but then OA would just see it as missing and put a second copy in the root again which I was hoping to avoid.

**Reply by advplyr** [reply]
What exactly are you doing to get this error?

**Reply by johnhwright** [reply]
Synced the series from OA, noticed ABS didn't have the first in the series but had the other 4. 
Moved the first book's mb4 out of the library directory (waited for the auto scan to finish). Put the file back in the directory, the auto scan runs and puts that error in the log and the book isn't added. Is that error just a red herring and there's something else stopping the book from showing up?

**Reply by advplyr** [reply]
I have no clue what happened but I would suggest removing the book from Abs by using the delete. Uncheck the box so it doesn't delete the actual file. Then rescan the library

![image](https://github.com/advplyr/audiobookshelf/assets/67830747/2150dcc6-94d0-427c-91e5-ddc8c9e30a07)
